Lab1/Server/Itit_2/Itit_2/Program.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+
+namespace Itit_2
+{
+    class Program
+    {
+            static readonly object _lock = new object();
+            static readonly Dictionary<int, TcpClient> list_clients = new Dictionary<int, TcpClient>();
+
+            static void Main(string[] args)
+            {
+                int count = 1;
+
+                TcpListener ServerSocket = new TcpListener(IPAddress.Any, 5000);
+                ServerSocket.Start();
+
+                while (true)
+                {
+                    TcpClient client = ServerSocket.AcceptTcpClient();
+                    lock (_lock) list_clients.Add(count, client);
```

Comment:
Consider using ConcurrentDictionary object instead of Dictionary and lock combination.

Reply:
Made.